components/financial_assistance/app/models/financial_assistance/applicant.rb

```diff
@@ -1618,6 +1618,18 @@ def individual_market_eligibility
       @individual_market_eligibility = eligibilities.where(_type: 'Eligibilities::V3::IndividualMarketEligibility').first
     end
 
+    # Builds a new APTC/CSR eligibility for the applicant.
+    #
+    # @param applicant [FinancialAssistance::Applicant] The applicant for whom the eligibility is being built.
+    # @return [Eligibilities::V3::AptcCsrEligibility] The newly built eligibility.
+    def build_aptc_csr_eligibility
+      eligibilities.build(
+        _type: 'Eligibilities::V3::AptcCsrEligibility',
```

Comment:
Not sure if this works without `include Mongoid::Attributes::Dynamic` on eligibility super class. Also if it does not working without the extension. This will likely be a bug in future. Since we will have remove the extension after the migration.

Reply:
I verified the type, it is working. This is not required `include Mongoid::Attributes::Dynamic` in the code.  I will also included in spec coverage.